src/main/java/dev/sigstore/rekor/client/RekorClient.java

```diff
@@ -15,18 +15,21 @@
  */
 package dev.sigstore.rekor.client;
 
-import com.google.api.client.http.ByteArrayContent;
-import com.google.api.client.http.GenericUrl;
-import com.google.api.client.http.HttpRequest;
-import com.google.api.client.http.HttpResponse;
+import com.google.api.client.http.*;
 import dev.sigstore.http.HttpProvider;
+import dev.sigstore.json.GsonSupplier;
 import java.io.IOException;
 import java.net.URI;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
 
 /** A client to communicate with a rekor service instance. */
 public class RekorClient {
   public static final String PUBLIC_REKOR_SERVER = "https://rekor.sigstore.dev";
   public static final String REKOR_ENTRIES_PATH = "/api/v1/log/entries";
+  public static final String REKOR_ENTRIES_GET_PATH = "/api/v1/log/entries/";
```

Comment:
presumably you already tried this, but there's no way to dedup this?

Reply:
yeah I can just construct the path.